### PR TITLE
Language selector: abbreviation text links instead of select box

### PR DESCRIPTION
**Pull request by kandre**
This PR change select box for language change to links on desktop (see #2394). Mobile behavior stay unchanged.

[Test link](https://mf-geoadmin3.dev.bgdi.ch/kan_lang_change_links/)

### CODE REVIEW

File: src/components/translation/partials/translation.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<select ng-model="lang" ng-options="l.value as l.label for l in options.langs">
-</select>
+<select ng-if="isMobile" ng-model="$parent.lang" ng-options="l.value as l.label for l in options.langs"></select>
+<a href ng-if="!isMobile" ng-class="$parent.lang == l.value ? 'ga-selected-link' : ''" ng-click="$parent.selectLang(l.value)" ng-repeat="l in options.langs" style="margin-left:5px">{{ l.label }}</a>
```
Comment (oterral) [inline src/components/translation/partials/translation.html]: style in the .less file

**Comment by oterral** [comment]
the template of the mobile and desktop version are totally different so maybe you can use 2 different partials and avoid using ng-if and $parent

**Comment by cedricmoullet** [comment]
Do we really want that ? (Cd bund)

Envoyé de mon iPhone

**Comment by gjn** [comment]
[swisstopo.admin.ch](http://www.swisstopo.admin.ch/) has it that way. [admin.ch](https://www.admin.ch/gov/de/start.html) has it that way (both in slightly different colors though)

**Comment by cedricmoullet** [comment]
ok, right. I thought it was a combo. So ok !

2015-06-10 8:55 GMT+02:00 Gilbert Jeiziner <notifications@github.com>:

> swisstopo.admin.ch <http://www.swisstopo.admin.ch/> has it that way.
> admin.ch <https://www.admin.ch/gov/de/start.html> has it that way (both
> in slightly different colors though)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoadmin/mf-geoadmin3/pull/2406#issuecomment-110619213>
> .
>



-- 
Twitter: http://twitter.com/cedricmoullet
Linked In: http://www.linkedin.com/in/cedricmoullet

<http://twitter.com/cedricmoullet>

**Comment by cedricmoullet** [comment]
@kandre could you add some space between the choices ?

**Comment by kandre** [comment]
Yes for sure, it seems that it broke somewhere

**Comment by cedricmoullet** [comment]
jenkins test

**Comment by oterral** [comment]
jenkins test

**Comment by kandre** [comment]
@oterral I will have to change ng-class ternary condition syntax

**Comment by kandre** [comment]
Changes done

**Comment by cedricmoullet** [comment]
jenkins test

**Comment by oterral** [comment]
THx @kandre nice work

**Pull request merged**